cmd/terraform_native_commands.go

```diff
@@ -0,0 +1,201 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// getTerraformCommands returns an array of statically defined Terraform commands with flags
+func getTerraformCommands() []*cobra.Command {
+	// List of Terraform commands
+	return []*cobra.Command{
+		{
+			Use:   "plan",
+			Short: "Show changes required by the current configuration",
+			Long:  "Generate an execution plan, which shows what actions Terraform will take to reach the desired state of the configuration.",
+		},
+		{
+			Use:   "apply",
+			Short: "Apply changes to infrastructure",
+			Long:  "Apply the changes required to reach the desired state of the configuration. This will prompt for confirmation before making changes.",
+		},
+		{
+			Use:   "workspace",
+			Short: "Manage Terraform workspaces",
+			Long:  "Create, list, select, or delete Terraform workspaces, which allow for separate states within the same configuration.",
+		},
+		{
+			Use:   "clean",
+			Short: "Clean up resources",
+			Long:  "Remove unused or outdated resources to keep the infrastructure clean and reduce costs.",
+		},
+		{
+			Use:   "version",
+			Short: "Show the current Terraform version",
+			Long:  "Displays the current version of Terraform installed on the system.",
+		},
+		{
+			Use:   "varfile",
```

Comment:
This should be in a new section for Atmos specific commands

Reply:
Same as for `clean`